src/views/MainPage/MainPage.js

```diff
@@ -1,19 +1,28 @@
-import Button from '../../components/Button/Button.js';
 import './MainPage.css';
+import { retrieveAnimalFact } from '../../api/FactsController.js';
+import { Fact } from '../../model/Fact.js';
+import Button from '../../components/Button/Button.js';
 import { onNavigationChange } from '../../components/router/Router.js';
 
-const MAIN_ANIMAL_PATH = './kangoroo.png';
-const FACT_ANIMAL_URL = 'http://placekitten.com/400/500';
-const ADOPTION_ANIMAL_URL = 'http://placekitten.com/300/400';
+const MAIN_ANIMAL = { path: './kangoroo.png', alt: 'kangoroo that tells informations' };
+const FACT_ANIMAL = {
+  url: 'http://placekitten.com/400/500',
+  alt: 'the animal the sentence is about',
+  fallbackFact:
+    "Cat's eyes shine in the dark because of the tapetum, a reflective layer in the eye, which acts like a mirror.",
+};
```

Comment:
We have it in the Fact class - it is needless here

Reply:
I was going to do that after refactoring (as above)